src/components/Firestore/index.tsx

```diff
@@ -147,9 +142,17 @@ export const Firestore: React.FC<FirestoreProps> = ({ config, projectId }) => {
   );
 };
 
-export const FirestoreRouteLoading: React.FC = () => (
-  // TODO
-  <div>Loading...</div>
+export const FirestoreLoading: React.FC = () => (
+  <GridCell
+    span={12}
+    className="Firestore-loading"
+    data-testid="firestore-loading"
+  >
+    <CircularProgress size="xlarge" />
+    <Typography use="body2" tag="p">
+      Firestore Emulator Loading...
+    </Typography>
```

Comment:
This and the centering styles are reused, can you extract into a component?

In console we have `<fire-spinner message="Blah loading" />`, where the default is to show a spinner only. or include the message if provided.

I think something like that would be helpful.

Reply:
And we could use without the grid and centering stuff with `<Spinner inline size="xsmall" />`

Perhaps you could even include the Grid and span={}.. its often going to be included as a grid flow: `<Spinner span={12} />`

Reply:
The extra component stuff is easy to do later, but making it into a simple component `<Spinner message="test" />` is helpful now.

Reply:
Added the grid version now but I'll leave the `inline` version unimplemented until we actually need it.